setup.py

```diff
@@ -81,7 +81,7 @@ def get_data_dirs(path, patterns):
             'psycopg2==2.5.4',
             'numpy==1.9.1',
             'python-daemon==2.0.5',
-            'ipython==0.13.0',
+            'ipython==3.1.0',
```

Comment:
Is this required? I'm keeping the IPython version low because we do quite a bit of monkey patching of IPython. I'll try out locally